packages/utils/src/internals/error_tracker.ts

```diff
@@ -1,9 +1,13 @@
 import { inspect } from 'node:util';
 
+import type { CrawlingContext } from '@crawlee/basic';
```

Comment:
same here

Reply:
Do you mean import it from core?
`import type { CrawlingContext } from '@crawlee/core';`

Reply:
Should I move both [`error_tracker.ts`](https://github.com/apify/crawlee/pull/2332/files/5f50e30181dfd3e680e2d82c373bdd5e1e619d06#diff-4d1d859f54527510499d11cbc93b6a48973df6cda5fbfdd72a0e28e80cb5ff08) and [`error_snapshotter.ts`](https://github.com/apify/crawlee/pull/2332/files/5f50e30181dfd3e680e2d82c373bdd5e1e619d06#diff-055eb16e2322f653218a8ba71a433a368d5314eb194c80edb6c016178cd5d5c8) classes to core as both need `CrawlingContext`?